docs/framework/windows-workflow-foundation/samples/usage-of-the-switch-activity-with-custom-types.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: "Usage of the Switch Activity with Custom Types"
 ms.custom: ""
-ms.date: "03/30/2017"
+ms.date: "11/08/2017"
 ms.prod: ".net-framework"
 ms.reviewer: ""
 ms.suite: ""
 ms.tgt_pltfrm: ""
 ms.topic: "article"
 ms.assetid: 482a48c4-eb83-40c3-a4e2-2f9a8af88b75
 caps.latest.revision: 10
-author: "Erikre"
-ms.author: "erikre"
-manager: "erikre"
+author: "jralexander"
+ms.author: "johalex"
+manager: "wpickett"
 ---
 # Usage of the Switch Activity with Custom Types
-This sample describes how to enable a <!--zz <xref:System.Activities. Statements.Switch`1>--> `xref:System.Activities` Statements.Switch`1?qualifyHint=False&autoUpgrade=True activity to evaluate a user-defined complex type at runtime. In most traditional procedural programming languages, a [switch](http://go.microsoft.com/fwlink/?LinkId=180521) statement selects an execution logic based on the conditional evaluation of a variable. Traditionally, a `switch` statement operates on an expression that can be statically evaluated. For example, in C# this means that only primitive types, such as <xref:System.Boolean>, <xref:System.Int32>, <xref:System.String>, and enumeration types are supported.  
+This sample describes how to enable a <xref:System.Activities.Statements.Switch> activity to evaluate a user-defined complex type at runtime. In most traditional procedural programming languages, a [switch](http://go.microsoft.com/fwlink/?LinkId=180521) statement selects an execution logic based on the conditional evaluation of a variable. Traditionally, a `switch` statement operates on an expression that can be statically evaluated. For example, in C# this means that only primitive types, such as <xref:System.Boolean>, <xref:System.Int32>, <xref:System.String>, and enumeration types are supported.  
```

Comment:
The link is broken, It should be \<xref:System.Activities.Statements.Switch%601>.